public/scripts/roles.js

```diff
@@ -0,0 +1,25 @@
+class Role {
+  constructor(image, name, description) {
+    this.image = image
+    this.name = name
+    this.description = description
+  }
+}
```

Comment:
I'm trying to avoid classes and the `new` operator to construct objects in JS now that I've been enlightened, but if you like the way this works I'm fine with it. If you're curious as to the motivation behind not using "classes", see https://github.com/getify/You-Dont-Know-JS/blob/master/this%20%26%20object%20prototypes/ch6.md#chapter-6-behavior-delegation